### PR TITLE
feat: implement queues and redis configuration AB#24208

### PR DESCRIPTION
**Pull request by Vladimir-Petkov**

### CODE REVIEW

File: services/.env.example [hunk 1]
```diff
@@ -153,3 +153,6 @@ MOCK_COMMERCIAL_BANK_ETHIOPIA=
 
 # Boolean to sync data automatically with third parties (now only used for Intersolve Visa)
 SYNC_WITH_THIRD_PARTIES=
+
+REDIS_HOST=
+REDIS_PORT=
```
Comment (elwinschmitz) [inline services/.env.example:158]: Can this be added somewhere higher up? (Close to other relevant ENV-variables)
+ Add some explanation in comments above it?

File: services/docker-compose.development.yml [hunk 2]
```diff
@@ -47,5 +47,10 @@ services:
       - '121_service_node_modules:/home/node/app/node_modules'
       - './121-service:/home/node/app'
 
+  121-redis:
+    image: 'redis:alpine'
```
Comment (elwinschmitz) [inline services/docker-compose.development.yml:51]: No version-number?

We can only use the one that is supported by the Azure service we'd use, right?

File: services/121-service/package.json [hunk 3]
```diff
@@ -43,6 +43,7 @@
   "private": true,
   "dependencies": {
     "@nestjs/axios": "^3.0.0",
+    "@nestjs/bull": "^10.0.1",
```
Comment (elwinschmitz) [inline services/121-service/package.json:46]: Can we also add some relevant links to the README of the 121-service?

**Comment by jannisvisser** [comment]
closing this because now integrated in #4549

**Pull request closed without merging**